templates/index.html

```diff
@@ -3,9 +3,37 @@
 <head>
     {% include "includes/header.html" %}
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>jacob.covs.tld</title> <!-- TEMP:REPLACE -->
```

Comment:
We'll need to make some config file that constants like this get pulled from. OK for now